tools/shared.py

```diff
@@ -1497,6 +1502,9 @@ def has_substr(array, substr):
 
   @staticmethod
   def configure(args, stdout=None, stderr=None, env=None):
+    # Check once and cache results
+    check_sanity()
```

Comment:
these also seem unnecessary?

Reply:
Hmm, according to comments, `emconfigure` (this `configure` function is its main function, yes?) performs builds with regular `clang`... Maybe it is really unnecessary here. On the other hand, would it take significant amount of time to check sanity once in case some day `emcc` will eventually be used instead of `clang` (meanwhile, is native `clang` invoked directly or proxyed by `emcc.py` now?).